scripts/Run_uncrustify_for_sources.bat

```diff
@@ -0,0 +1,91 @@
+@echo off
+setlocal
+
+rem
+rem 14.12.2016
+rem
+rem Checks if all source files conform
+rem to the uncrustify configuration
+rem The check is performed with the uncrustify
+rem program that is available in the path.
+rem This might be an older version than
+rem a newly build one.
+rem 
+rem Call this batch script from the top level
+rem directory of the uncrustify project.
+
+set SRC_DIR=src
+set OUT_DIR=results
+set CFG_FILE=forUncrustifySources.cfg
+set DIFF_FILE=lastdiff.txt
+set "PWD=%cd%"
+
+rem Check if uncrustify is available in path
```

Comment:
Thanks for picking to up. I was thinking about it too.

I think there are two use cases:
1. Checking if a changed uncrustify code corresponds to the expected format -> this is for uncrustify developers
2. Checking if the code of any other project corresponds to the expected format -> this is for uncrustify users

For 1. it might be more appropriate to use the newly build uncrustify binary. Even tough this might fail if in the new binary introduced a new bug. So there is also a good reason to use an older uncrustify version that is know to work.

For 2. Usually there is no uncrustify source project present but an uncrustify binary. So in this case searching the path seems more appropriate to me.

The best solution would probably be to use a command line parameter for the script to specifiy which uncrustify binary to use. If the parameter is missing the path is searched as fallback. 

I think we should propose the scripts Run_uncrustify_for_sources.bat/sh to uncrustify users that simply want to check there own projects. Maybe adding a comment to README.md?